### PR TITLE
Fix authentication

### PR DESCRIPTION
**Pull request by tarzzz**
@n-riesco What we discussed on slack.. 
This is needed for fixing on-prem authentication.

  - [x] Tested on on-prem

### CODE REVIEW

File: backend/plugins/authorization.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 import fetch from 'node-fetch';
 
 const ESCAPED_ROUTES = [
-  new RegExp(`^${homeUrl()}/$`),
-  new RegExp(`^${homeUrl()}/ping$`),
-  new RegExp(`^${homeUrl()}/login$`),
-  new RegExp(`^${homeUrl()}/static/`),
-  new RegExp(`^${homeUrl()}/oauth2$`),
-  new RegExp(`^${homeUrl()}/oauth2/callback$`)
+  new RegExp('^/$'),
```
Comment (n-riesco) [inline backend/plugins/authorization.js]: Could you add a comment explaining why we mustn't prefix the RegExp with `${homeUrl}`, please?

Comment (n-riesco) [reply backend/plugins/authorization.js]: Otherwise, we'll make this mistake again.

Comment (tarzzz) [reply backend/plugins/authorization.js]: Added.. 💃 💃 ?

**Comment by n-riesco** [comment]
:dancer:

**Pull request merged**